velox/exec/tests/CrossJoinTest.cpp

```diff
@@ -195,7 +195,8 @@ TEST_F(CrossJoinTest, zeroColumnBuild) {
   };
 
   auto rightVectors = {
-      makeRowVector({sequence<int32_t>(5)}),
+      makeRowVector({sequence<int32_t>(1)}),
+      makeRowVector({sequence<int32_t>(4, 1)}),
       //      vectorMaker_.rowVector(ROW({}, {}), 5),
```

Comment:
While you are here, would you remove this commented out code?

Reply:
Done.